src/main/java/com/epam/ta/reportportal/personal/PersonalProjectService.java

```diff
@@ -120,7 +120,7 @@ public static ProjectConfiguration defaultConfiguration(Project project) {
 		defaultConfig.setInterruptJobTime(InterruptionJobDelay.ONE_DAY.getValue());
 		defaultConfig.setKeepLogs(KeepLogsDelay.THREE_MONTHS.getValue());
 		defaultConfig.setKeepScreenshots(KeepScreenshotsDelay.TWO_WEEKS.getValue());
-		defaultConfig.setProject(project);
+		project.setConfiguration(defaultConfig);
```

Comment:
I am not sure it makes sense to inject default config into project here, because same operation is done in line #95, where this method is called:
```
project.setConfiguration(defaultConfiguration(project));
```

Reply:
You're right. It's totally useless set. Removed.